### PR TITLE
Create Neural-Artistic-Style-Transfer.yml

### PR DESCRIPTION
**Pull request by riya-17**

### CODE REVIEW

File: _data/projects/Artistic-Neural-Style-Transfer.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Artistic Neural Style Transfer
```
Comment (ritwik12) [inline _data/projects/Artistic-Neural-Style-Transfer.yml]: @riya-17 Your project name is not same as your repo name, is it fine?

File: _data/projects/Artistic-Neural-Style-Transfer.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+desc: Artistic Neural Style Transfer helps to transfer the style of your favourite artist style image to any content image.
+site: https://github.com/riya-17/Neural-Artistic-Style-Transfer
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- Tensorflow
+- python
+- numpy
```
Comment (ritwik12) [inline _data/projects/Artistic-Neural-Style-Transfer.yml]: @riya-17 You can add more tags like Neural network, Scipy, Machine learning and many more.

File: _data/projects/Artistic-Neural-Style-Transfer.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+upforgrabs:
+  name: good first issue
+link: https://github.com/riya-17/Neural-Artistic-Style-Transfer/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (ritwik12) [inline _data/projects/Artistic-Neural-Style-Transfer.yml]: Please correct the indentation for link and add your label link instead of issue link

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Artistic-Neural-Style-Transfer.yml (hunk 1 above)
Comment (riya-17) [reply _data/projects/Artistic-Neural-Style-Transfer.yml]: I have changed it. Thanks

File: _data/projects/Artistic-Neural-Style-Transfer.yml (hunk 2 above)
Comment (riya-17) [reply _data/projects/Artistic-Neural-Style-Transfer.yml]: ok thanks

File: _data/projects/Artistic-Neural-Style-Transfer.yml (hunk 3 above)
Comment (riya-17) [reply _data/projects/Artistic-Neural-Style-Transfer.yml]: ok thanks

**Comment by ritwik12** [comment]
@riya-17  Please change your Project name in PR name also

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@riya-17 Thanks!!

**Comment by riya-17** [comment]
@ritwik12 Thanks :)

**Pull request merged**